src/controllers/RecipeController.js

```diff
@@ -58,6 +58,12 @@ const addRecipe = asyncHandler(async (req, res) => {
     res.status(400);
     throw new Error('Body missing from request!');
   }
+  if (!verify(req.headers.authorization)) {
+    res.status(400);
+    throw new Error(
+      'Unauthorized access detected! Only the appropriate user can add this!'

```

Comment:
'Unauthorized access detected! Please log in before adding a recipe'?

Reply:
done